homotopy-core/src/common.rs

```diff
@@ -51,12 +51,20 @@ pub type SingularHeight = usize;
 pub type RegularHeight = usize;
 
 #[derive(PartialEq, Eq, Copy, Clone, Debug, Hash)]
-#[cfg_attr(feature = "fuzz", derive(arbitrary::Arbitrary))]
 pub enum Height {
     Singular(SingularHeight),
     Regular(RegularHeight),
 }
 
+#[cfg(feature = "fuzz")]
+impl<'a> arbitrary::Arbitrary<'a> for Height {
```

Comment:
Can't this be derived?

Reply:
Yes, and it originally was. I changed it to avoid having to spend more time debugging overflow errors which are likely to never show up in reality. For example, when we convert from the Singular/Regular heights indexing into usize, we multiple by 2, which can in theory cause overflow, but it's never actually gonna be a problem and also it's most definitely not work fixing everywhere.

The value I set here is high enough to be able to cause out of bounds error wherever they haven't been fixed.